docs/4.x/debugging-strategies.md

```diff
@@ -0,0 +1,52 @@
+---
+category: DevelopInDepth
+---
+
+This document lists strategies that can be useful in debugging the many different parts of Matomo while developing on the platform.
+You might find these useful if you are contributing to core or are writing your own plugin.
+
+# Debugging Tracking Code
```

Comment:
could we name this `Debugging the JS tracker`? because there is already debugging the tracker guide https://developer.matomo.org/api-reference/tracking-api#debugging-the-tracker

Reply:
@tsteur modified